openshift/template.yaml

```diff
@@ -414,126 +332,82 @@ objects:
             timeoutSeconds: 10
           volumeMounts:
             - mountPath: /results
-              name: pv-analyzes
+              name: analyzes
         restartPolicy: Always
         volumes:
-          - name: pv-analyzes
+          - name: analyzes
             persistentVolumeClaim:
-              claimName: claim1
+              claimName: analyzes
     test: false
     triggers:
     - type: ConfigChange
     - type: ImageChange
       imageChangeParams:
         automatic: true
         containerNames:
-          - thoth-result-api
-        from:
-          kind: ImageStreamTag
-          name: 'thoth-result-api:latest'
-
-- apiVersion: v1
-  kind: DeploymentConfig
-  metadata:
-    labels:
-      service: graphexp
-      app: graphexp
-    name: graphexp
-    namespace: ${THOTH_FRONTEND_NAMESPACE}
-  spec:
-    replicas: 1
-    selector:
-      service: graphexp
-    template:
-      metadata:
-        labels:
-          service: graphexp
-      spec:
-        containers:
-        - name: thoth-graphexp
-          image: thoth-graphexp
-          env:
-            - name: THOTH_JANUSGRAPH_HOST
-              value: janusgraph-hostname-${THOTH_MIDDLEEND_NAMESPACE}.${THOTH_ROUTING_SUFFIX}
-          ports:
-            - containerPort: 8080
-              protocol: TCP
-          readinessProbe:
-            httpGet:
-              path: /index.html
-              port: 8080
-            initialDelaySeconds: 10
-            periodSeconds: 10
-            timeoutSeconds: 10
-        restartPolicy: Always
-    triggers:
-    - type: ConfigChange
-    - type: ImageChange
-      imageChangeParams:
-        automatic: true
-        containerNames:
-          - thoth-graphexp
+          - result-api
         from:
           kind: ImageStreamTag
-          name: 'thoth-graphexp:latest'
+          name: 'result-api:stable'
 
 #
 # Persistent volume claims
-#
 - apiVersion: v1
   kind: PersistentVolumeClaim
   metadata:
-    name: claim1
-    namespace: ${THOTH_MIDDLEEND_NAMESPACE}
+    name: analyzes
   spec:
     accessModes:
       - ReadWriteMany
     resources:
       requests:
-        storage: 16Gi
+        storage: 10Gi
+    storageClassName: ${STORAGE_CLASS}
 
 - apiVersion: v1
   kind: PersistentVolumeClaim
   metadata:
-    name: claim2
-    namespace: ${THOTH_FRONTEND_NAMESPACE}
+    name: buildlogs
   spec:
     accessModes:
       - ReadWriteMany
     resources:
       requests:
-        storage: 16Gi
+        storage: 10Gi
+    storageClassName: ${STORAGE_CLASS}
 
 #
 # Cron jobs
-#
 - apiVersion: batch/v2alpha1
   kind: CronJob
   metadata:
-    name: thoth-cleanup-job
-    namespace: ${THOTH_FRONTEND_NAMESPACE}
+    name: cleanup-job
   spec:
     #schedule: "@weekly"
     schedule: "*/1 * * * *"
     jobTemplate:
       spec:
         template:
           spec:
-            serviceAccountName: master-access
+            serviceAccountName: analyzer
```

Comment:
The cleanup job is not analyzer so maybe this is confusing.

Reply:
right, who uses it for what?

Reply:
It is used by the user API service to run pods on analysis request as well as by the cleanup-job that cleans old analyzes [1].

https://github.com/fridex/thoth-core#the-overall-architecture

Reply:
both use cases seem to be related to an analyzer, that is why I called the SA "analyzer" :P

Reply:
It's not assigned to analyzer itself, but rather to the other parts that work with analyzers - that's why it sounds confusing to me. Something like analyzer-control or stuff like that would trigger more meaning to "something that handles things with analyzers".